shell/browser/extensions/api/pdf_viewer_private/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+import("//extensions/buildflags/buildflags.gni")
+import("//pdf/features.gni")
+
+assert(enable_extensions,
+       "Cannot depend on extensions because enable_extensions=false.")
+assert(enable_pdf)
+
+source_set("pdf_viewer_private") {
+  sources = [
+    "pdf_viewer_private_api.cc",
+    "pdf_viewer_private_api.h",
+  ]
+
+  deps = [
```

Comment:
This file is not required given the sources are already included as part of `lib_sources_extensions` above and these deps should go under https://github.com/electron/electron/blob/a2d35e9cf53f324d8153cb56982074e1af5fb177/BUILD.gn#L739